components/Link/index.tsx

```diff
@@ -0,0 +1,42 @@
+import NextLink from "next/link";
+import Icon from "../Icon";
+import styles from "./styles.module.css";
+
+interface LinkProps extends React.ComponentProps<typeof NextLink> {
+  children: React.ReactNode;
+  internal?: boolean;
+  external?: boolean;
```

Comment:
Firstly, is there any reason why you're allowing manual overrides of internal/external? When is there a reason to not make the regex handle this?

Secondly (although I would opt to remove both if they are not needed), why define both internal and external? I normally feel like one would be the opposite of the other..

Reply:
Removed them as an option now :)